v-next/hardhat/test/internal/cli/init/project-creation.ts

```diff
@@ -0,0 +1,8 @@
+import { describe, it } from "node:test";
+
+describe("project-creation", function () {
+  it.todo(
```

Comment:
added a "todo" test to check that the latest sol version is used in the hardhat.config.ts file. Currently, V3 has no function to check this